src/request.ts

```diff
@@ -152,6 +165,97 @@ export async function getRequestLinkDetails({
 	return x
 }
 
+export async function prepareXchainRequestLinkFulfillmentTransaction({
+	senderAddress,
+	recipientAddress,
+	destinationChainId,
+	destinationToken,
+	fromAmount,
+	link,
+	squidRouterUrl,
+	provider,
+	apiUrl = 'https://api.peanut.to/',
```

Comment:
todo: is this consistent with how we name this variable in other places? please make sure. I dont know if its baseurl or apiurl, please check